ext/tokenizer/tests/token_get_all_variation15.phpt

```diff
@@ -49,6 +49,7 @@ var_dump( token_get_all($source));
 echo "Done"
 ?>
 --EXPECTF--
+Deprecated: Directive 'short_open_tag' is deprecated in Unknown on line 0
```

Comment:
Should be able to drop it from this test, it only uses `<?=`.

Reply:
While I'm at it, shouldn't this be a ``--EXPECT--`` instead of the current ``--EXPECTF--``?

Reply:
It uses `int(%d)` so no, this has to stay EXPECTF.

Reply:
Oh, totally missed those